spread.yaml

```diff
@@ -270,6 +270,38 @@ suites:
                 apt-get purge -y snapd
             fi
 
+    tests/completion/:
```

Comment:
so i could inherit the environ down into the tests (otherwise i wold've had to duplicate it).

Also, they're very different and weird :-)